.travis.yml

```diff
@@ -0,0 +1,18 @@
+if: branch = master and type IN (pull_request)
+language: cpp
+services:
+  - docker
+env:
+before_install:
+  - docker pull fiware/orion-ci:rpm7
+install: true
+before_script: true
+script: true
+jobs:
+  include:
+    - stage: unit
```

Comment:
If we were able to use the regular docker for this (see https://github.com/telefonicaid/fiware-orion/pull/3192/files#r191405706) docker images would be changed:

fiware/orion-ci:rpm7 -> fiware/orion:latest

Reply:
We can easily change image for ci docker image to fiware/orion: with tag ("ci"), except alternative repo fiware/orion-ci. But for now it's better to have them both.

Reply:
Yes, I agree.

Reply:
NTC